frontend/src/metabase/modes/components/drill/ActionClickDrill/utils.ts

```diff
@@ -58,25 +65,13 @@ function isMappedParameter(
 
 export function getNotProvidedActionParameters(
   action: WritebackAction,
-  parameterMappings: ActionParametersMapping[],
-  mappedParameters: ParameterMappedForActionExecution[],
+  dashboardParamValues: ParameterMappedForActionExecution[],
 ) {
-  const emptyParameterTargets: ParameterTarget[] = [];
-
-  mappedParameters.forEach(mapping => {
-    if (mapping.value === undefined) {
-      emptyParameterTargets.push(mapping.target);
-    }
-  });
```

Comment:
we don't need to check for undefined here because prepareParameter() already filters those out.